### DISCUSSION TITLE
annotation files

### DISCUSSION THREAD

**Discussion by e-coral** [Ruitulyu/KAS-Analyzer#11] [Q&A]
How were the annotation files collected/created, please?

I ask this question because I am attempting to use KAS-Analyzer with a t2t reference genome. So far, I have generated the relevant indexed reference genome, and added the relevant chromosome sizes and blacklist files. These have allowed me to perform many KAS-Analyzer functions. However, I am now attempting to run functions that require annotation files, and it is not clear to me how these were obtained/generated for other versions of the genome. I have attempted to download refseq genes in bed format via the UCSC table browser, but the file isn't in the same format as the hg38 refseq genes file within KAS-Analyzer. 

Do you have any advice or suggestions for how I can generate the required annotation bed files for t2t?

**Comment by Ruitulyu** [comment]
Hi,
I will try to include the t2t reference genome in KAS-Analyzer very
shortly.
Best,
Ruitu

On Mon, Nov 27, 2023 at 3:56 AM E Croot ***@***.***> wrote:

> How were the annotation files collected/created, please?
>
> I ask this question because I am attempting to use KAS-Analyzer with a t2t
> reference genome. So far, I have generated the relevant indexed reference
> genome, and added the relevant chromosome sizes and blacklist files. These
> have allowed me to perform many KAS-Analyzer functions. However, I am now
> attempting to run functions that require annotation files, and it is not
> clear to me how these were obtained/generated for other versions of the
> genome. I have attempted to download refseq genes in bed format via the
> UCSC table browser, but the file isn't in the same format as the hg38
> refseq genes file within KAS-Analyzer.
>
> Do you have any advice or suggestions for how I can generate the required
> annotation bed files for t2t?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Ruitulyu/KAS-Analyzer/discussions/11>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APDPVGBHUMYI5C43ACDGMTLYGRPV5AVCNFSM6AAAAAA73XX56OVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHA4TKOJTGY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Nutricula, based in Chicago.

**Comment by e-coral** [comment]
Thank you very much for adding relevant t2t files to the tool
